tests/processing/test_process_global_results.py

```diff
@@ -1,5 +1,5 @@
 
-from glambie.processing.process_global_results import _combine_regional_results_into_global
+from glambie.processing.process_global_results import _combine_regional_results_in_mwe_into_global
```

Comment:
No tests for new functions. The new functions should have at least some cursory coverage.